hips/utils/tests/test_healpix.py

```diff
@@ -22,3 +25,44 @@ def test_boundaries():
     radec_precomp = [[264.375,  258.75,  264.375,  270.],
                      [-24.624318,  -30.,  -35.685335,  -30.]]
     assert_allclose([radec.ra, radec.dec], radec_precomp)
+
+def test_compute_image_pixels():
+    order = 3
+    nside = hp.order2nside(order)
+
+    hdu = fits.open('https://github.com/gammapy/gammapy-extra/blob/master/datasets/catalogs/fermi/gll_psch_v08.fit.gz?raw=true')
```

Comment:
Avoid doing web queries from tests whenever possible. They are slower and have more spurious fails than tests that just do computations and local file I/O.

Like I mentioned in the last telcon, my suggestion would be that you add a utility function to this package that generates a WCS object from scratch using a few parameters.
You can do this here or in a separate branch / PR first.

Example is here:
http://astropy.readthedocs.io/en/latest/wcs/index.html#building-a-wcs-structure-programmatically

Example how to write a function that takes a few parameters and returns a WCS object is here:
https://github.com/gammapy/gammapy/blob/master/gammapy/maps/wcs.py#L138

For this (and then later tile drawing) tests, I'd suggest to have something that doesn't cover all-sky, but maybe you could do something like this:
* AIT or CAR projection
* 1500 x 800 pixels
* Galactic coordinates with reference point at the center of the image and (GLON=10 deg, GLAT = 5 deg)